apps/judicial-system/web/src/routes/Court/Indictments/CourtRecord/CourtSessionAccordionItem.tsx

```
@@ -919,6 +980,106 @@ const CourtSessionAccordionItem: FC<Props> = (props) => {
                 </Box>
               </Box>
             </MultipleValueList>
+            {mergedCaseDocumentsInCourtSession.map((mergeDocuments) => {
+              const courtCaseNumber = mergeDocuments.courtCaseNumber
+              const mergedFiledDocuments = mergeDocuments.mergedFileDocuments
+              return (
+                <Box>
+                  <SectionHeading
+                    title={`Dómsskjöl úr sameinuðu máli ${courtCaseNumber}`}
+                  />
+                  <BlueBox>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)